netlify/functions/manage-findings.ts

```diff
@@ -4,57 +4,69 @@ import { Event } from "@netlify/functions/src/function/event";
 import { Octokit } from "@octokit/core";
 import { createOrUpdateTextFile } from "@octokit/plugin-create-or-update-text-file";
 
-
-import { Finding, FindingEditRequest, FindingResponse, FindingsResponse } from "../../types/findings";
+import {
+  Finding,
+  FindingEditRequest,
+  FindingsResponse,
+} from "../../types/finding";
+import { Contest } from "../../types/contest";
 
 import { checkAuth } from "../util/auth-utils";
 import { getContest, isContestActive } from "../util/contest-utils";
-import { getAvailableFindings, wardenFindingsForContest } from "../util/github-utils";
-import { getUserTeams } from "../util/user-utils";
+import {
+  getAvailableFindings,
+  wardenFindingsForContest,
+} from "../util/github-utils";
+import {
+  getUserTeams,
+  checkAndUpdateTeamAddress,
+  sendConfirmationEmail,
+} from "../util/user-utils";
+
+import { apiKey, domain } from "../_config";
 
 async function getFinding(
   username: string,
-  contestId: number,
   issueId: number,
+  contest: Contest
```

Comment:
Converting to an object is probably good, but maybe move it back above issueId?

It's kind of an ordering of contexts.

Reply:
Oh, sure. Good point.